python/python/lance/fragment.py

```diff
@@ -164,6 +165,10 @@ def create(
             *Experimental API*. Progress tracking for writing the fragment. Pass
             a custom class that defines hooks to be called when each fragment is
             starting to write and finishing writing.
+        mode: str, default "append"
```

Comment:
Should you mention what other modes are available?  Will there ever be a distinction between `create` and `overwrite`?  If not, should this just be a boolean column?

Reply:
Are you thinking something like `check_existing_schema: bool=True`?

Reply:
I feel like "append" vs "overwrite" and "create" communicate how to use this in terms that already exist in our APIs. I don't really foresee any difference between "overwrite" and "create".

Reply:
I was thinking something like `overwrite_existing_schema` with a default of False but I don't feel too strongly if you like append.

Reply:
Yeah I think I like this as append for now.